templates/strings/objc-h.stencil

```diff
@@ -0,0 +1,66 @@
+// Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
+
+{% if tables.count > 0 %}
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+{% macro parametersBlock types %}{% filter removeNewlines:"leading" %}
+  {% for type in types %}
+    ({% call paramTranslate type %})p{{ forloop.counter }}{% if not forloop.last %} :{% endif %}
+  {% endfor %}
+{% endfilter %}{% endmacro %}
+{% macro argumentsBlock types %}{% filter removeNewlines:"leading" %}
+  {% for type in types %}
+    p{{forloop.counter}}{% if not forloop.last %}, {% endif %}
+  {% endfor %}
+{% endfilter %}{% endmacro %}
+{% macro paramTranslate swiftType %}
+  {% if swiftType == "Any" %}
+    id
+  {% elif swiftType == "CChar" %}
+    char
+  {% elif swiftType == "Float" %}
+    float
+  {% elif swiftType == "Int" %}
+    NSInteger
+  {% elif swiftType == "String" %}
+    id
+  {% elif swiftType == "UnsafePointer<CChar>" %}
+    char*
+  {% elif swiftType == "UnsafeRawPointer" %}
+    void*
+  {% else %}
+    objc-h.stencil is missing '{{swiftType}}'
+  {% endif %}
+{% endmacro %}
+{% macro emitOneMethod table item %}
+{% for string in item.strings %}
+{% if not param.noComments %}
+// {{string.key}} --> "{{string.translation}}"
+{% endif %}
+{% if string.types %}
+  {% if string.types.count == 1 %}
++ (NSString*){{string.key|swiftIdentifier:"pretty"|lowerFirstWord}}WithValue:{% call parametersBlock string.types %}
```

Comment:
Doesn't this (and the following one) method declaration stencil need a `;` at the end? I tried using this stencil in my project and it doesn't compile because of missing `;` after every method declaration.

Reply:
i agree, it's missing the semi-colon. i probably fixed the stencil in my local project and forgot to update this repo.

Reply:
fixed by [PR 672](https://github.com/SwiftGen/SwiftGen/pull/672).